### PR TITLE
initial commit on dataTable sorting

### PR DESCRIPTION
**Pull request by sdellis**

### CODE REVIEW

File: src/elements/DataTable.vue [hunk 1]
```diff
@@ -7,11 +7,25 @@
     </caption>
     <thead>
       <tr>
-        <th v-for="(col, index) in parsedColumns" scope="col">{{ displayName(col) }}</th>
+        <th v-for="(col, index) in parsedColumns" scope="col">
+          <lux-icon-base v-if="col.sortable" width="16" height="16" icon-name="approved">
```
Comment (carolyncole) [inline src/elements/DataTable.vue]: I'm confused by the icon name here.  Sould sort be better than approved?

File: src/elements/DataTable.vue [hunk 2]
```diff
@@ -126,6 +149,29 @@ export default {
         return col.name
       }
     },
+    sortTable(value) {
+      if (!value.ascending) {
+        this.rows.sort(function(a, b) {
+          var textA = a[value.name.toLowerCase()].toString().toLowerCase()
+          var textB = b[value.name.toLowerCase()].toString().toLowerCase()
```
Comment (carolyncole) [inline src/elements/DataTable.vue]: Does this work for 35 and 5?  Aka 5 is less than 35 numerically, but is likely to be greater than 35 in a string comp.

File: test/unit/specs/components/datatable.spec.js [hunk 3]
```diff
@@ -15,8 +15,8 @@ describe("DataTable.vue", () => {
         columns: [
           { name: "id", display_name: "Select Items", align: "center", checkbox: true },
           "name",
-          { name: "email", display_name: "Email Address", align: "center" },
-          { name: "age", datatype: "number", summary_value: "33" },
+          { name: "email", display_name: "Email Address", align: "center", sortable: true },
+          { name: "age", datatype: "number", summary_value: "33", sortable: true },
         ],
         jsonData: [
           { id: 1, name: "foo", email: "foo@xxx.xxx", age: 42 },
```
Comment (carolyncole) [inline test/unit/specs/components/datatable.spec.js]: Change this line to 4 and see where is sorts to validate my numeric sort question

File: src/elements/DataTable.vue (hunk 1 above)
Comment (sdellis) [reply src/elements/DataTable.vue]: Nice catch, @carolyncole! That's an artifact from an early experiment. I'll fix.

File: src/elements/DataTable.vue (hunk 2 above)
Comment (sdellis) [reply src/elements/DataTable.vue]: Again, nice catch. This only does alphanumeric. We will need to test the type for this, but the previous code I had will work for numeric.

**Pull request merged**